packages/client/src/app/services/user/my-stuff.service.ts

```diff
@@ -29,8 +29,20 @@ export class MyStuffService {
         }), catchError(err => {
           return throwError(err);
         }));
-    } else if (kind === ContentKind.WorkContent) {
-      // fetch work
+    } else if (kind === ContentKind.ProseContent) {
+      return this.http.get<ProseContent>(`${this.url}/fetch-one?contentId=${contentId}&kind=${kind}`, {observe: 'response', withCredentials: true})
+        .pipe(map(res => {
+          return res.body;
+        }), catchError(err => {
+          return throwError(err);
+        }));
```

Comment:
this little block feels pretty boilerplatey. Not something for this PR, but a good candidate for wrapping up into a little helper function, I think